reflect/src/main/java/dagger/reflect/UnlinkedMapOfProviderBinding.java

```diff
@@ -13,11 +13,10 @@
   }
 
   @Override
-  public LinkedBinding<Map<Object, Provider<Object>>> link(Linker linker, Scope scope) {
```

Comment:
This is unfortunate. Not that I expect anyone to ever see these types, and they're erased anyway, but it at least forces a bit of safety into the method. Doing `LinkedBinding<Map<?, Provider<?>>>` doesn't work?